### PR TITLE
Enable bitcode for targets

### PR DESCRIPTION
**Pull request by russelarms**
Enable bitcode for targets so PLCR can be successfully used as a submodule.

### CODE REVIEW

File: CrashReporter.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3813,6 +3814,7 @@
 				HEADER_SEARCH_PATHS = "\"$(SRCROOT)/Dependencies/protobuf-c\"";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				MACOSX_DEPLOYMENT_TARGET = 10.7;
+				OTHER_CFLAGS = "-fembed-bitcode";
```
Comment (MatkovIvan) [inline CrashReporter.xcodeproj/project.pbxproj:3817]: Is there separate setting for it? Let's avoid usage `OTHER_CFLAGS` when possible

Comment (russelarms) [reply CrashReporter.xcodeproj/project.pbxproj]: Changed to BITCODE_GENERATION_MODE.

**Comment by russelarms** [comment]
>Update readme, if this flags is not required in build command anymore

I'm going to test and change it after the testing.

**Comment by MatkovIvan** [comment]
Please also pay attention to
https://github.com/microsoft/plcrashreporter/blob/1256cc417984031f65cdf7248a8151aa3a4c3433/Scripts/build-ios-universal.sh#L20
and similar places

**Comment by russelarms** [comment]
I reverted to use `Other C Flags` as `BITCODE_GENERATION_MODE` doesn't allow us to remove bitcode flags from build scripts.

**Pull request merged**